aria/__init__.py

```diff
@@ -28,6 +28,9 @@
 from .VERSION import version as __version__
 from .storage.drivers import ResourceDriver, ModelDriver, FileSystemModelDriver, FileSystemResourceDriver
 from .storage import ModelStorage, ResourceStorage, models
+from .storage.storage_utils import (create_deployment_storage,
```

Comment:
remove from storage, lets add "application_tools" module (aria.application_tools)
In there we can write the "create_deployment_plan_storage" func.
lets change the name - create_deployment_plan_storage => two func's
1. fill_storage_with_blueprint - lets think of a better name :-)
    This func will put:
        1. node model,
        2. Blueprint model,
        3. Relationship model,
        4. Plugin model,
        5. blueprint resource,
        6. plugin resource
2. fill_storage_with_deployment - lets think of a better name :-)
    This func will put:
        1. Deployment model,
        2. NodeInstance model,
        3. RelationshipInstance model,
        5. deployment resource,